### ISSUE TITLE
Regular Expression Parsing

### ISSUE THREAD

**Issue by nareshshah139** [TeamHG-Memex/eli5#197]
Team ELI5: 

Some possible issues with respect to regular expressions that I encountered (and fixed for myself). Perhaps other users will face similar issues. 

1.  '^(\t*)(\d+):\[(\w+)<([^\]]+)\] '
        'yes=(\d+),no=(\d+),missing=(\d+),'
        'gain=([^,]+),cover=(.+)$'

Is the regular expression to parse the xgboost dump. This works well on say the titanic variable names but if a feature name has a ' . ' character in the feature name, it results in a formatting error. Perhaps this should be noted and documented. ' . ' character in the feature name is a common practice to denote derived feature names. I've changed my derived feature name to be using _ instead of the ' .  ' character now.

**Comment by nareshshah139** [comment]
![regular_expression_mismatch](https://cloud.githubusercontent.com/assets/1490119/26139753/7f1f53fe-3af0-11e7-8276-c1612bedff24.PNG)
![regular_expression_working](https://cloud.githubusercontent.com/assets/1490119/26139758/892e9968-3af0-11e7-8b51-9a57391e8002.PNG)

**Comment by kmike** [comment]
A good catch @nareshshah139! I've released 0.6.2 with a fix; could you please check that it works for you?

**Comment by nareshshah139** [comment]
Works well now! Thanks!
